tests/kernel/install_ltp.pm

```diff
@@ -271,5 +272,10 @@ will be used.
 
 Overrides the official LTP GitHub repository URL.
 
+=head2 GRUB_PARAM
```

Comment:
I wonder is this documentation somewhere rendered? Any doc server or something?

Reply:
I have no idea.

Reply:
I really doubt this is published, but @okurz generating and publishing in http://open.qa would be a nice idea (Also for the rest of the test distribution... but I don't know if it's really needed, but I do see a benefit).